src/python_framework_v02/troute/nhd_network_augment.py

```diff
@@ -716,128 +746,156 @@ def main():
-        
+
         print("merging segments...")
         data_merged, qlat_destinations = segment_merge(
-                                    data,
-                                    data_snapped, 
-                                    network_data, 
-                                    threshold, 
-                                    pruned_segs
-                                   )
-        
+            data, data_snapped, network_data, threshold, pruned_segs
+        )
+
     if not snap and prune:
         dirname = "RouteLink_" + supernetwork + "_" + str(threshold) + "m_prune_merge"
-        filename = "RouteLink_" + supernetwork + "_" + str(threshold) + "m_prune_merge.shp"
-        filename_cw = "CrossWalk_" + supernetwork + "_" + str(threshold) + "m_prune_merge.json"
-        
+        filename = (
+            "RouteLink_" + supernetwork + "_" + str(threshold) + "m_prune_merge.shp"
+        )
+        filename_cw = (
+            "CrossWalk_" + supernetwork + "_" + str(threshold) + "m_prune_merge.json"
+        )
+
         print("Prune and merge:")
         print("snapping junctions...")
         data_snapped = snap_junctions(data, threshold, network_data)
-        
+
         print("merging segments...")
         data_merged, qlat_destinations = segment_merge(
-                                    data,
-                                    data_snapped, 
-                                    network_data, 
-                                    threshold, 
-                                    pruned_segs
-                                   )
-        
+            data, data_snapped, network_data, threshold, pruned_segs
+        )
+
     if not snap and not prune:
         dirname = "RouteLink_" + supernetwork + "_" + str(threshold) + "m_merge"
         filename = "RouteLink_" + supernetwork + "_" + str(threshold) + "m_merge.nc"
-        filename_cw = "CrossWalk_" + supernetwork + "_" + str(threshold) + "m_merge.json"
+        filename_cw = (
+            "CrossWalk_" + supernetwork + "_" + str(threshold) + "m_merge.json"
+        )
         print("Just merge:")
         print("merging segments...")
         data_merged, qlat_destinations = segment_merge(
-                                    data,
-                                    data, 
-                                    network_data, 
-                                    threshold, 
-                                    pruned_segs
-                                   )
-    
+            data, data, network_data, threshold, pruned_segs
+        )
+
     # update RouteLink data
-    RouteLink_edit = RouteLink.loc[data_merged.index.values]    
+    RouteLink_edit = RouteLink.loc[data_merged.index.values]
 
-    for (columnName, columnData) in data_merged.iteritems(): 
-        RouteLink_edit.loc[:,columnName] = columnData
+    for (columnName, columnData) in data_merged.iteritems():
+        RouteLink_edit.loc[:, columnName] = columnData
 
     for idx in RouteLink_edit.index:
```

Comment:
An operation like this is much better expressed in pandas as:
```python
RouteLink_edit["to"] = RouteLink_edit["to"].clip(lower=0)
```
We avoid the explicit iterating thru the index which will be slow.

Reply:
@awlostowski-noaa Please make a note of this as an issue and we can address it in a separate PR.